base/java-tools/src/com/netscape/cmstools/KRATool.java

```diff
@@ -4872,6 +4891,18 @@ public static void main(String[] args) {
             mKraNamingContextMessage = "";
         }
 
+        // Check for the Key Unwrap Algorithm provided by user.
+        // If unprovided, choose DES3 as the default (to maintain consistency with old code)
+        if (keyUnwrapAlgorithmName == null || keyUnwrapAlgorithmName.equalsIgnoreCase("DES3")) {
+            mKeyUnwrapAlgorithm = SymmetricKey.DES3;
+        } else if (keyUnwrapAlgorithmName.equalsIgnoreCase("AES")) {
+            mKeyUnwrapAlgorithm = SymmetricKey.AES;
```

Comment:
I am not sure. I created this since old KRAs used DES3 and the new KRAs use AES. I am planning to rewrite the tool once I get enough cycle since there are a lot of hardcoded values and is not future proof :)

Reply:
Since the tool wasn't in a working condition this is probably OK to check in, but please check with @jmagne or @vakwetu later to see which algorithms the tool should support.

Reply:
I had a chat with Jack and he asked me to take a look at CRMFPopClient to list the supported algorithms. I see only DES3 and AES to be supported.

https://github.com/dogtagpki/pki/blob/DOGTAG_10_5_BRANCH/base/java-tools/src/com/netscape/cmstools/CRMFPopClient.java#L226